### ISSUE TITLE
Draft, Approve, Translate Copy for College Discount

### ISSUE THREAD

**Issue by vykster** [cal-itp/benefits#1099]
## Copy & Translation Process:

- [x] #1102
- [x] #1103 @vykster _in progress_
- [ ] #1100 @MarikoLochridge _in progress_
- [x] **Monday 11/7** #1101 led by @srhhnry 
- [x] **Tuesday 11/8** Copy handed off to Haley by @srhhnry  
- [x] **Wednesday 11/9** Copy received by @srhhnry from Haley, proofed and edited 
- [x] **Wednesday 11/9** Copy handed off by @srhhnry to Product, Engineering and Design for sign-off https://docs.google.com/document/d/10STeotEOeJubvu1nV6fpThcW49NfdjAuLBkDgaY721E/edit
- [x] **Tuesday 11/15** Copy needs to be signed off by Product, Engineering and Design by this date
- [ ] Engineers/Design may also decide to make changes to existing copy. If so, make sure these existing copy changes are part of the package handed off to the Translation team.
- [ ] **Translation hand-off:** @machikoyasuda to hand off approved copy to translation team, @machikoyasuda to hand off any style guides/glossaries/term banks to Translation team - tracked here https://github.com/cal-itp/benefits/issues/1121
- [ ] **Receive translation:** @machikoyasuda to receive translated copy from Translation team - tracked here https://github.com/cal-itp/benefits/issues/1121

Approved and translated copy is now ready for Design and Engineering.

## Design & Engineering Process:

The following can now happen after the copy approval process is done:
- Design will have proofed, edited and approved copy to use in Figma designs. #1110 
- Once designs are in, Engineers can begin implementing the copy in with the new user interfaces.
- Engineers/Design may also decide to make changes to existing copy. This can happen in parallel to design, before college implementation begins.

## Post-process:
- [ ] **Post-process documentation** Add updated info on content localization process and resources to Services Team Handbook @machikoyasuda

**Comment by vykster** [comment]
From @srhhnry 

Questions we have as we put together a facilitation deck:
Who are we designing for? (personas)
Need clarification on whether this flow is for students/teachers/admin etc
We talked about a few sort-of ecosystem items that product can contribute to the facilitation deck--want to follow up on what those things are. Examples we discussed in person:
Transit language
Familiar transit signage
Other administrative processes

**Comment by afeld** [comment]
Given that CSUMB will include faculty and staff, have been referring to this as the "college discount".

**Comment by afeld** [comment]
> Who are we designing for? (personas) Need clarification on whether this flow is for students/teachers/admin etc

The discount offer varies by school:

- CSUMB: students, faculty, and staff
- Harnell: students
- MPC: students

The schools likely won't launch at the same time.

**Comment by machikoyasuda** [comment]
Latest status:

Copy has been drafted by a team (led by @srhhnry, assited by @afeld @mrose914 @MarikoLochridge @machikoyasuda), and then proofed and edited by Haley.

Copy has been handed off to Design, Product and Engineering for approval on Wed. Nov. 9th: https://docs.google.com/document/d/10STeotEOeJubvu1nV6fpThcW49NfdjAuLBkDgaY721E/edit

Design, Product and Engineering must approve and sign-off on copy by Tues. Nov. 15th by the latest.

**Comment by vykster** [comment]
@machikoyasuda are you willing to handle the handoff to the content localization vendor?

**Comment by vykster** [comment]
@machikoyasuda apologies I see you update the issue description, thank you!

**Comment by machikoyasuda** [comment]
STATUS: There are a few Google Doc comment threads unresolved remaining.

STATUS: This whole milestone is being de-prioritized at the moment, as the Data Sharing Agreement has not yet been signed.

**Comment by vykster** [comment]
Broke out translation work into new issue, #1189 Closing this one as complete
